app/models/service_template_transformation_plan/validate_config_info.rb

```diff
@@ -14,18 +14,32 @@ def validate_config_info(options)
                 else
                   config_info[:transformation_mapping]
                 end
-
+      # mapping.invalid? added as a part of validating migration plan
       raise _('Must provide an existing transformation mapping') if mapping.blank?
+      raise _('Transformation mapping was invalid.  Provide a valid one') if mapping.invalid?
```

Comment:
Do you get the reason why the mapping is invalid ? If yes, it would be great to add it to the error message.

Reply:
Yes, adding it now.